tests/ftqc/test_pauli_tracker.py

```diff
@@ -50,36 +50,124 @@ def test_xz_decode_pauli_unsupported_error(self, x, z):
     @pytest.mark.parametrize(
         "ops, expected",
         [
-            ([qml.I(0)], qml.I(0)),
-            ([qml.X(1)], qml.X(1)),
-            ([qml.Y(2)], qml.Y(2)),
-            ([qml.Z(3)], qml.Z(3)),
-            ([qml.I(0), qml.I(0)], qml.I(0)),
-            ([qml.I(1), qml.X(1)], qml.X(1)),
-            ([qml.I(2), qml.Y(2)], qml.Y(2)),
-            ([qml.I(3), qml.Z(3)], qml.Z(3)),
-            ([qml.X(0), qml.I(0)], qml.X(0)),
-            ([qml.X(1), qml.X(1)], qml.I(1)),
-            ([qml.X(2), qml.Y(2)], qml.Z(2)),
-            ([qml.X(3), qml.Z(3)], qml.Y(3)),
-            ([qml.Y(0), qml.I(0)], qml.Y(0)),
-            ([qml.Y(1), qml.X(1)], qml.Z(1)),
-            ([qml.Y(2), qml.Y(2)], qml.I(2)),
-            ([qml.Y(3), qml.Z(3)], qml.X(3)),
-            ([qml.Z(0), qml.I(0)], qml.Z(0)),
-            ([qml.Z(1), qml.X(1)], qml.Y(1)),
-            ([qml.Z(2), qml.Y(2)], qml.X(2)),
-            ([qml.Z(3), qml.Z(3)], qml.I(3)),
-            ([qml.X(4), qml.Y(4), qml.Z(4), qml.I(4), qml.Z(4)], qml.Z(4)),
+            ([qml.I(0)], (0, 0)),
+            ([qml.X(1)], (1, 0)),
+            ([qml.Y(2)], (1, 1)),
+            ([qml.Z(3)], (0, 1)),
+            ([qml.I(0), qml.I(0)], (0, 0)),
+            ([qml.I(1), qml.X(1)], (1, 0)),
+            ([qml.I(2), qml.Y(2)], (1, 1)),
+            ([qml.I(3), qml.Z(3)], (0, 1)),
+            ([qml.X(0), qml.I(0)], (1, 0)),
+            ([qml.X(1), qml.X(1)], (0, 0)),
+            ([qml.X(2), qml.Y(2)], (0, 1)),
+            ([qml.X(3), qml.Z(3)], (1, 1)),
+            ([qml.Y(0), qml.I(0)], (1, 1)),
+            ([qml.Y(1), qml.X(1)], (0, 1)),
+            ([qml.Y(2), qml.Y(2)], (0, 0)),
+            ([qml.Y(3), qml.Z(3)], (1, 0)),
+            ([qml.Z(0), qml.I(0)], (0, 1)),
+            ([qml.Z(1), qml.X(1)], (1, 1)),
+            ([qml.Z(2), qml.Y(2)], (1, 0)),
+            ([qml.Z(3), qml.Z(3)], (0, 0)),
+            ([qml.X(4), qml.Y(4), qml.Z(4), qml.I(4), qml.Z(4)], (0, 1)),
```

Comment:
Nice work on this extensive test set, but I'm wondering if there is a way to generate these programmatically that is more maintainable that this expansive list